docs/moderator-handbook.md

```diff
@@ -190,7 +190,11 @@ Again, if you have any problems or questions, make a post with your actions in t
 
 ### Dealing with Posted Solutions
 
-When a user posts a solution to any of the questions, remove it and either, kindly let them know that solutions are not permitted on the forum in your own words or, use the **Solution Instead of Help** "Canned reply".
+When a user replies in a help thread specific to the freeCodeCamp curriculum with a solution, remove it and either, kindly let them know that solutions are not permitted in help threads in your own words or, use the **Solution Instead of Help** "Canned reply".
+
+If the OP (Original Poster) replies within a curriculum help thread with their final solution, blur it and use the **Blurred Spoiler Solution** "Canned reply".
+
+If a user creates a topic asking for feedback on a solution, that topic should be moved to the [feedback subforum] if it is not already there and the solution should be blurred. But if the user posts the solution to show if off, unlist the topic.
```

Comment:
```suggestion
If a user creates a thread asking for feedback on a solution, move the thread to the feedback subforum and the blur the solution, as necessary. If the user is only posting the solution to show if off, then unlist the thread and use the **Solutions Thread** canned reply.
```

Reply:
There is a small mistake in the suggestion here "and the blur".

I am also not sure of the meaning of "as necessary". Do we need that phrase?

Reply:
'As necessary' or 'if needed' - it might be in the right sub forum and already blurred

Reply:
"blur the solution" -> "spoiler the solution"?

Reply:
The button on the forum says 'blur spoiler' for me and that's what the canned reply says, that that's why I picked that